test/Conversion/MooreToCore/basic.mlir

```diff
@@ -77,6 +77,55 @@ func.func @Expressions(%arg0: !moore.bit, %arg1: !moore.logic, %arg2: !moore.pac
   %4 = moore.mir.shr %arg2, %arg0 : !moore.packed<range<bit, 5:0>>, !moore.bit
   %5 = moore.mir.shr arithmetic %arg2, %arg2 : !moore.packed<range<bit, 5:0>>, !moore.packed<range<bit, 5:0>>
   %6 = moore.mir.shr arithmetic %arg3, %arg2 : !moore.packed<range<bit<signed>, 4:0>>, !moore.packed<range<bit, 5:0>>
+  
+  // CHECK: comb.add %arg0, %arg0 : i1
+  // CHECK: comb.sub %arg0, %arg0 : i1
+  // CHECK: comb.mul %arg0, %arg0 : i1
+  // CHECK: comb.divu %arg0, %arg0 : i1
+  // CHECK: comb.modu %arg0, %arg0 : i1
+  // CHECK: comb.and %arg0, %arg0 : i1
+  // CHECK: comb.or %arg0, %arg0 : i1
+  // CHECK: comb.xor %arg0, %arg0 : i1
+  %7 = moore.add %arg0, %arg0 : !moore.bit
+  %8 = moore.sub %arg0, %arg0 : !moore.bit
+  %9 = moore.mul %arg0, %arg0 : !moore.bit
+  %10 = moore.div %arg0, %arg0 : !moore.bit
+  %12 = moore.mod %arg0, %arg0 : !moore.bit
+  %13 = moore.and %arg0, %arg0 : !moore.bit
+  %14 = moore.or %arg0, %arg0 : !moore.bit
+  %15 = moore.xor %arg0, %arg0 : !moore.bit
+  
+  // CHECK: comb.icmp ult %arg0, %arg0 : i1
+  // CHECK: comb.icmp ule %arg0, %arg0 : i1
+  // CHECK: comb.icmp ugt %arg0, %arg0 : i1
+  // CHECK: comb.icmp uge %arg0, %arg0 : i1
```

Comment:
Cool! Can you also add test cases for a signed type, like `!moore.bit<signed>`, to check that they produce `slt/sle/sgt/sge`?

Reply:
The slang will return `!moore.bit` when using relational and equality ops to handle two `!moore.bit<signed>`. So should use operands to estimate to choose `ult` or `slt`.

Reply:
It is already added. Thanks! 😺

Reply:
Awesome! Thanks :partying_face: